### PR TITLE
VPN-7644: fix some CI tests

### PR DESCRIPTION
**Pull request by mcleinman**
## Description

Most of the common CI test failures - 
- IP Info: having something hit wrong (should be fixed here)
- Addons: Has date off by a minute (should be fixed here)
- Addons: Unexpected shows "Yesterday". I'm not 100% sure on this one yet, it is not fixed here.
- MessagesAddonPopover: Sometimes times out partway through the run. I have a hopeful fix here, but I'm not confident it will do it.

## Reference

VPN-7644

## Checklist
    
- [x] My code follows the style guidelines for this project
- [x] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [x] I have performed a self review of my own code
- [x] I have commented my code PARTICULARLY in hard to understand areas
- [x] I have added thorough tests where needed

### CODE REVIEW

**Comment by mcleinman** [comment]
Well, functional tests all passed on first try, so that's an encouraging sign.

File: tests/functional/testMessageAddonPopover.js [hunk 1]
```diff
@@ -17,6 +17,10 @@ async function loadAddonsAndNavigateHome(testAddonName) {
       'Mozilla.Shared', 'MZAddonManager', 'loadCompleted', 'true');
 
   await vpn.waitForQueryAndClick(queries.navBar.HOME.visible());
+  await vpn.waitForQueryAndClick(
```
Comment (oskirby) [inline tests/functional/testMessageAddonPopover.js]: A common edge case that trips things up in functional testing is expecting a thing to be clickable once `visible()` is `true` but this often doesn't work whenever there are loading delays or property animations. The usual fix when I've run into these kind of test issues is to add a `ready` property that becomes `true` only after all the transitions and animations are finished.

Comment (mcleinman) [reply tests/functional/testMessageAddonPopover.js]: Thanks, updated.

**Review by oskirby — APPROVED** [review]
LGTM
